2-python-fundamentals-i.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
_Commented on notebook `2-python-fundamentals-i.ipynb` Cell 49 Line 6_
```markdown
## Numbers

Recall that numbers in python are represented by the `int` and `float` types. We can perform all standard numerical operations: 

* `x + y` : sum of `x` and `y`
* `x - y` : differenceof `x` and `y`
```
typos in this section